### DISCUSSION TITLE
建议新增生成文章二维码海报的功能用于分享到移动端

### DISCUSSION THREAD

**Discussion by Nonoas** [halo-dev/halo#6643] [Ideas]
### 你当前使用的版本

1.4.13

### 描述一下此特性

点击分享时，可以选择生成二维码海报图片，可以下载用于分享文章

**Comment by ruibaby** [comment]
感谢建议，但目前不考虑在系统内置这样的功能，可以等到插件系统完成后为此开发一个插件。

/milestone 2.0
/kind feature
